### ISSUE TITLE
Introspection fails when there are multiple foreign keys between two tables

### ISSUE THREAD

**Issue by aruntk** [prisma/prisma2#707]
prisma version -> prisma2@2.0.0-preview-13.2
db -> postgres 11.5

here is the table creation sql

```
-- SCHEMA: prisma_test

-- DROP SCHEMA prisma_test ;

CREATE SCHEMA prisma_test
    AUTHORIZATION postgres;

-- Table: prisma_test."User"

-- DROP TABLE prisma_test."User";

CREATE TABLE prisma_test."User"
(
    email text COLLATE pg_catalog."default" NOT NULL DEFAULT ''::text,
    id text COLLATE pg_catalog."default" NOT NULL,
    name text COLLATE pg_catalog."default",
    CONSTRAINT "User_pkey" PRIMARY KEY (id)
)
WITH (
    OIDS = FALSE
)
TABLESPACE pg_default;

ALTER TABLE prisma_test."User"
    OWNER to postgres;

-- Index: User.email

-- DROP INDEX prisma_test."User.email";

-- Table: prisma_test."Post"

-- DROP TABLE prisma_test."Post";

CREATE TABLE prisma_test."Post"
(
    content text COLLATE pg_catalog."default",
    "createdAt" timestamp(3) without time zone NOT NULL DEFAULT '1970-01-01 00:00:00'::timestamp without time zone,
    id text COLLATE pg_catalog."default" NOT NULL,
    published boolean NOT NULL DEFAULT true,
    title text COLLATE pg_catalog."default" NOT NULL DEFAULT ''::text,
    "updatedAt" timestamp(3) without time zone NOT NULL DEFAULT '1970-01-01 00:00:00'::timestamp without time zone,
    author text COLLATE pg_catalog."default",
    "coAuthor" text COLLATE pg_catalog."default",
    CONSTRAINT "Post_pkey" PRIMARY KEY (id),
    CONSTRAINT "Post_author_fkey" FOREIGN KEY (author)
        REFERENCES prisma_test."User" (id) MATCH SIMPLE
        ON UPDATE NO ACTION
        ON DELETE SET NULL,
    CONSTRAINT "Post_co_author_fkey" FOREIGN KEY ("coAuthor")
        REFERENCES prisma_test."User" (id) MATCH SIMPLE
        ON UPDATE NO ACTION
        ON DELETE NO ACTION
)
WITH (
    OIDS = FALSE
)
TABLESPACE pg_default;

ALTER TABLE prisma_test."Post"
    OWNER to postgres;
```

When I try to introspect this schema it throws an error at the second foreign key relationship (`coAuthor`). 

<img width="534" alt="Screenshot 2019-10-10 at 10 01 09 AM" src="https://user-images.githubusercontent.com/6007432/66539222-eee80f80-eb44-11e9-9e2d-ae034cda471e.png">

**Comment by pantharshit00** [comment]
I am able to reproduce this with `prisma2@2.0.0-alpha.232, binary version: 19ed9db8b704a89eef6fe710fa52c2ad0493443f`. Error message is different though.

![image](https://user-images.githubusercontent.com/22195362/66568702-9ab24f00-eb87-11e9-9b86-5cb218d173f6.png)

**Comment by pantharshit00** [comment]
Just released this issue is a duplicate of #699 

Thanks for reporting this anyways though :). Please post your further findings if any there from now.
